### PR TITLE
Master

### PR DESCRIPTION
**Pull request by frostblogNet**
Please make sure these boxes are checked before submitting your PR - thank you!

- [ ] Pull request is based against develop branch
- [ ] README.md reflects changes (if applicable)
- [ ] New files contain a license header

### Issue

This PR fixes issue # .

### Proposed changes

...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,23 +16,18 @@ env:
...
-    env: MAGENTO_VERSION=2.2.* CODE_QUALITY=true
+    env: MAGENTO_VERSION=2.2.*
   - php: 7.2
     env: MAGENTO_VERSION=2.3.0
   - php: 7.2
     env: MAGENTO_VERSION=2.3.*
-
+  - php: 7.3
+    env: MAGENTO_VERSION=2.3.0 CODE_QUALITY=true
+  - php: 7.4
```
Comment (sprankhub) [inline .travis.yml]: Hmm Magento does not support PHP 7.4 yet, right? Does that still make sense?

Comment (frostblogNet) [reply .travis.yml]: Thats true this was just a tipo. I have fixed it. But I still have the problem that I cannot run the travis ci test. Hope we can fix this to get the version ready.

File: .travis.yml [hunk 2]
```diff
@@ -16,23 +16,18 @@ env:
...
   - php: 7.1
-    env: MAGENTO_VERSION=2.2.* CODE_QUALITY=true
+    env: MAGENTO_VERSION=2.2.*
   - php: 7.2
     env: MAGENTO_VERSION=2.3.0
   - php: 7.2
     env: MAGENTO_VERSION=2.3.*
-
+  - php: 7.3
+    env: MAGENTO_VERSION=2.3.0 CODE_QUALITY=true
```
Comment (sprankhub) [inline .travis.yml]: Not sure what the `CODE_QUALITY` flag does, but should this again be on the `.*` env?

Comment (frostblogNet) [reply .travis.yml]: The flag will tell this pipeline to run the coding standard tests like phpcs and magento ecg code style checks. See here
https://github.com/firegento/firegento-magesetup2/blob/develop/Test/script/travis_script.sh#L13

**Pull request closed without merging**